src/app/shared-services/trajectory/trajectory-import-export.service.ts

```diff
@@ -41,6 +45,7 @@ export class TrajectoryImportExportService extends TrajectoryService {
    * it is for debug-purposes only and must be 'false' when pushed remotely.
    */
   private static importAsUserTrajectory = false
+  private trajectoryId: string
```

Comment:
I might be overlooking something, but is the `trajectoryId` needed somewhere else apart from `createTrajectoryFromImport` (in l. 113) ?

Otherwise it could simply be a const in l. ~128, instead of a class-wide variable:
`const trajectoryId = example ? 'example' : uuid()`

Reply:
It is only needed here, so I made the change